README.md

```diff
@@ -20,15 +20,18 @@ Two alternatives to TensorFlow's distribution strategies are available:
 
 ### Install with Pip
 
+Note that in order to support both tensorflow cpu and gpu in a single package, tf-yarn doesn't depend directly on tensorflow.
+You can either run `pip install tf-yarn` then install tensorflow or tensorflow_gpu, or use `pip install tf-yarn[cpu]` to do it all in one command.
+
 ```bash
-$ pip install tf-yarn
+$ pip install tf-yarn[cpu]
 ```
 
 ### Install from source
 
 ```bash
 $ git clone https://github.com/criteo/tf-yarn
-$ cd tf-yarn
+$ cd tf-yarn[cpu]
```

Comment:
We don't have any directory tf-yarn[cpu]

Reply:
indeed, typo, fixed